### ISSUE TITLE
Text description on mouse hover over icons

### ISSUE THREAD

**Issue by noenfugler** [Superalgos/Superalgos#304]
I'm finding it a bit hard to work out what many of the icons mean.  I've watched videos, and checked documentation.  Unless I'm looking in the wrong place, they seem to be undocumented. 

**Describe the solution you'd like**
Text description of icon when hovering on icon.
Expanded documentation with descriptions of what each icon is.

**Describe alternatives you've considered**
Two alternatives provided above.

**Comment by Luis-Fernando-Molina** [comment]
Good idea, and not so difficult to implement. I think I will include it on a sprint I have in mind where I want to better integrate with the documentation.

**Comment by Tentacule23** [comment]
If this was about the left menu icons, it's done.
